### PR TITLE
Invert the order of the table. 

### PR DESCRIPTION
**Pull request by its-VinayKumar**
**What kind of change does this PR introduce?**

**Issue Number:**
#572 

**Screenshots/videos:**

Screenshot (215)
Screenshot (216)

**If relevant, did you update the documentation?**

**Summary**
Columns order Changed: Common Name, Published, Meta-schema identifiers, Metaschemas and Notes.

### CODE REVIEW

**Comment by its-VinayKumar** [comment]
@benjagm Please Review it and Merge the PR. :)

File: components/Layout.tsx [hunk 1]
```diff
@@ -369,6 +369,9 @@ const Footer = () => (
         </div>
       </div>
     </div>
+    <div className='copyright-section'>
+      <p>Copyright © 2024 JSON Schema. All rights reserved.</p>
```
Comment (benjagm) [inline components/Layout.tsx:373]: Why this change is part of the PR?

**Review by benjagm — CHANGES_REQUESTED** [review]
With invert the order I meant the chronological order in how the drafts appear in each row the table. Currently it from older to newest/current and it is supposed to be the opposite, from newest/current to older.

File: components/Layout.tsx (hunk 1 above)
Comment (foad-khan) [reply components/Layout.tsx:373]: He should learn git first!

Comment (its-VinayKumar) [reply components/Layout.tsx:373]: Are you mentor of this organization and open source contribution is about collaborating and learning. :)

**Comment by benjagm** [comment]
Clased as the issue as already been fixed.
